### ISSUE TITLE
Wrong presentedView origin y

### ISSUE THREAD

**Issue by ishall1** [slackhq/PanModal#154]
### Description

Describe your issue here.

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [x] bug
- [ ] enhancement (feature request)
- [ ] question
- [ ] documentation related
- [ ] testing related
- [ ] discussion

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackhq/PanModal/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

---

### Bug Report

Filling out the following details about bugs will help us solve your issue sooner.

#### Reproducible in:

PanModal version:

iOS version:

#### Steps to reproduce:

1.
2.
3.

#### Expected result:

What you expected to happen

#### Actual result:

What actually happened

#### Attachments:

Logs, screenshots, screencast, sample project, funny gif, etc.

![image](https://user-images.githubusercontent.com/8748505/130029084-977d32ac-5407-47ea-aa36-37a04e6186ce.png)
I found shortFormYPosition longFormYPosition panFrame.origin.y are all Float, so the underlined code my case wrong result and presentedView.frame.origin.y may be wrong.

**Comment by patthehuman** [comment]
this project is dead my dude

**Comment by ishall1** [comment]
> this project is dead my dude

It's a sad story

**Comment by ppamorim** [comment]
I am maintaing this project here: https://github.com/ppamorim/PanModal
Could you please describe the issue better and a example of the issue? I don't understand your description.

**Comment by ishall1** [comment]
> I am maintaing this project here: https://github.com/ppamorim/PanModal
> Could you please describe the issue better and a example of the issue? I don't understand your description.

Float value is not accurate。

**Comment by patthehuman** [comment]
@ppamorim have you added any new features to your branch? I'm also making some contributions on my own private branch. Wondering if you did anything that we can merge?

**Comment by patthehuman** [comment]
@ishall1 youre right. points on the screen should be represented as CGfloat. However, CGFloat can be initted from Float so I suspect its prbably fine.

**Comment by ppamorim** [comment]
@patthehuman Yes I did:

 - Changed the way that the borders are setup, it's not using masks anymore.
 - Fixed bug where `contentInset` was not supported.
 - You can close from the max height state now.
 - You can detect if it either closed by the user or by the `dismiss` function.
 - You can draw over the current context.
 
 I also applied swiftlint and removed some warnings.

**Comment by ppamorim** [comment]
@ishall1 I don't think it makes sense, please read this: https://stackoverflow.com/questions/1264924/whats-the-difference-between-using-cgfloat-and-float/1264943

There is no significant difference between CGFloat and Float. Also your assumption is wrong, both `shortFormYPosition` and `longFormYPosition` are CGFloat.

**Comment by patthehuman** [comment]
@ppamorim cool i made some good fixes too.. maybe we can merge and keep this project updated.

**Comment by patthehuman** [comment]
@ppamorim  I added code to customize frame width/height/offset for the dragger indicator that sits above the pan modal.

**Comment by ishall1** [comment]
> @ishall1 I don't think it makes sense, please read this: https://stackoverflow.com/questions/1264924/whats-the-difference-between-using-cgfloat-and-float/1264943
> 
> There is no significant difference between CGFloat and Float. Also your assumption is wrong, both `shortFormYPosition` and `longFormYPosition` are CGFloat.

My issue is that as CGFloat/Float is not accurate, when adjustPresentedViewFrame is called more than once, the layout of presentedViewController may be wrong. 
I resolved it like the follow in my project:
![image](https://user-images.githubusercontent.com/8748505/134792499-f4a44480-b343-4357-a1ab-a551cdbb1ff1.png)
